index.html

```diff
@@ -2,10 +2,200 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Portfolio</title>
     <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <nav>
+        <ul>
+          <li>
+            <a href="#about">About</a>
+          </li>
+          <li>
+            <a href="#work">Work</a>
+          </li>
+          <li>
+            <a href="./images/Resume.pdf" target="blank">Resume</a>
+          </li>
+          <li>
+            <a href="#contact">Contact</a>
```

Comment:
Does this work?  If you don't have a plan to connect it up, then I suggest you remove and replace with some contact info